configs/config.novacustom_v540tnx

```diff
@@ -1,4 +1,4 @@
-CONFIG_LOCALVERSION="v0.9.1"
+CONFIG_LOCALVERSION="v1.0.0-rc1"
```

Comment:
Why v1.0.0? We do not use v1.0.0 which would indicate production-ready FW.

Reply:
The alternative was v0.10.0 which I expect would break a lot of version number parsers, or v0.9.2 which does not reflect that there was a major change in coreboot base

Reply:
@miczyg1 what do you think the version number should be?

Reply:
v0.9.3

For MSI we did the same. We just bump the patch version regardless of base change

Reply:
@miczyg1 done in https://github.com/Dasharo/coreboot/pull/640/commits/9427f7a0709ca45a87c5e5911add2f9c104b9d71